modules/search/js/search-widget-filters-admin.js

```diff
@@ -140,19 +176,22 @@
 			trackAndBumpMCStats( 'changed_filter_count', eventArgs );
 		} );
 
-		widget.on( 'click', '.jetpack-search-filters-widget__controls .add', function( e ) {
+		// add filter button
+		widget.on( 'click', '.jetpack-search-filters-widget__add-filter', function( e ) {
 			e.preventDefault();
-			var closest = $( this ).closest( '.jetpack-search-filters-widget__filter' ),
-				clone = closest
-					.clone()
-					.attr( 'class', 'jetpack-search-filters-widget__filter' );
-
-			clone.find( 'input[type="number"]' ).val( defaultFilterCount );
-			clone.find( 'input[type="text"]' ).val( '' );
-			clone.find( 'select option:first-child' ).prop( 'selected', true );
-
-			clone.insertAfter( closest );
-			clone.find( 'input, textarea, select' ).change();
```

Comment:
We need a `change` event like this so that the widget knows it's in a "dirty" state and needs to be saved. Otherwise, when I add a filter, and only a filter, the blue button doesn't change or become enabled.

Reply:
Are you sure? This is why I invoke `wp.customize.state( 'saved' ).set( false );` below...

Reply:
I'm sure. The `wp.customize.state...` call seems to work in the customizer, but it doesn't work for the wp-admin UI